app/activity/[title]/page.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+import {
+  Heading,
+  Box,
+  Container,
+  useColorModeValue,
+  Flex,
+  Spacer,
+  Text,
+  Textarea,
+  Input,
+  FormControl,
+  FormLabel
+} from '@chakra-ui/react'
+import { FiChevronLeft, FiClock, FiMapPin, FiLink2 } from 'react-icons/fi'
+import { Carousel } from '@/components/carousel'
+import { Link } from '@/components/link'
+import { Header, Footer } from '@/components/navigation'
+import { TrashIcon, EditIcon } from '@/icons'
+import { customColors } from '@/theme/color'
+
+export default function ActivityDetails({ params }: { params: { title: string } }) {
```

Comment:
I think id is correct as param, not title? Because title can be duplicated

Reply:
That makes sense! I changed it to id!